### ISSUE TITLE
How to pass non string variable to packer from this provider

### ISSUE THREAD

**Issue by kholisrag** [toowoxx/terraform-provider-packer#10]
We have : 

lets say we have `noamd-clients-ami.pkr.hcl`, that contains

```hcl
...
variable "ami_regions" {
  type        = list(string)
  description = "A list of regions to copy the AMI to"
  default     = ["us-east-1", "eu-central-1", "ap-southeast-1"]
}

variable "ami_tags" {
  type = map(string)
  description = "Tags pass to Packer Builded's AMI"
}
...
...

source "amazon-ebs" "nomad_ami" {
  ...
  tags = var.ami_tags
}
```

then we have `main.tf` that contains : 

```hcl
resource "packer_image" "nomad_ami" {
  ...
  variables = {
    ...
    ami_regions      = var.ami_regions
    ami_tags           = local.tags
    ...
  }

  triggers = {
    packer_version          = data.packer_version.version.version
    files_hash              = data.packer_files.nomad_ami.files_hash
    ...
  }
}
```

question : 
1. did the current provider support non string hcl type in the `variables` ?
2. can we have packer build logs when running with this provider ?

**Comment by simaotwx** [comment]
Duplicate of #7 
Complex input types are not supported right now. Please use `jsonencode` and `jsondecode` as a workaround.

**Comment by simaotwx** [comment]
Implemented in 0.16.0
